concourse/pipelines/templates/resources/pxf6-gp6-el8-artifact-tpl.yml

```
@@ -0,0 +1,13 @@
+{# template for a pxf6 gp6 el8 build artifact resource definition.
```

in the old PXF resource file we capitalized these words
```suggestion
{# template for a PXF6 GP6 EL8 build artifact resource definition.
```